### ISSUE TITLE
Test suite failure

### ISSUE THREAD

**Issue by snoyberg** [typedrat/stb-image-redux#1]
```
Linking dist/build/stb-image-redux-tests/stb-image-redux-tests ...
> /tmp/stackage-build8/stb-image-redux-0.1.0.3$ dist/build/stb-image-redux-tests/stb-image-redux-tests

Data.STBImage
  image loader
    loads the first ten pixels correctly
    loads the same image in different formats FAILED [1]
  image writer
    works idempotently
    saves and reloads lossless images correctly across formats
    saves and reloads PNG files correctly

Failures:

  test/Data/STBImageSpec.hs:29: 
  1) Data.STBImage, image loader, loads the same image in different formats
       uncaught exception: IOException of type UserError (user error (Pattern match failure in do expression at test/Data/STBImageSpec.hs:30:13-21))

Randomized with seed 1871866945

Finished in 0.0442 seconds
5 examples, 1 failure
```

**Comment by typedrat** [comment]
Should be resolved by 73c4d76/v0.1.0.4.

**Comment by snoyberg** [comment]
Thanks!
